### DISCUSSION TITLE
Update to allow a single button for remote to send multiple codes to tune to a specific channel error

### DISCUSSION THREAD

**Discussion by blasco403** [GerryDazoo/Slinger#150] [General]
I tried to utilize the new functionality to allow for multiple commands to be sent with a single button via the remote. And yes I updated the code to the patched version released a few hours ago ;-)....

This is the text I have in the remote configuration txt file for my Sling500.

'Global' : 9,18,9,42

It should send 101 and also OK as this is what is required on my shaw (xfinity STB).

This is the error I get back on the server:

Starting remote control stream handler for  ('192.168.0.10', 55004) to port 58049
Remote Control Connected
Button Clicked ImmutableMultiDict([('Global', '9,18,9')]) 192.168.0.10
[2022-10-19 10:56:30,863] ERROR in app: Exception on /Remote/500 [POST]
Traceback (most recent call last):
  File "flask\app.py", line 2077, in wsgi_app
  File "flask\app.py", line 1525, in full_dispatch_request
  File "flask\app.py", line 1523, in full_dispatch_request
  File "flask\app.py", line 1509, in dispatch_request
  File "slingbox_server.py", line 1397, in button
    for keycode in data.split(','):
ValueError: invalid literal for int() with base 10: '9,18,9,42'
Exiting Remote Control Stream Handler for ('192.168.0.10', 55004)

**Comment by GerryDazoo** [comment]
I don't think your running the latest code. Please post ALL server output. Should be running 3.0.8c
Sorry Standby I may have missed a line

**Reply by GerryDazoo** [reply]
I fat fingered updating the exe file. Grab the latest exe file

**Comment by blasco403** [comment]
Ok, I will have a look again and test.

Thanks for getting back to me. 

TTYL

Ron

On Oct 19, 2022, at 12:36 PM, GerryDazoo ***@***.***> wrote:

﻿
I don't think your running the latest code. Please post ALL server output. Should be running 3.0.8c

**Reply by GerryDazoo** [reply]
I fat fingered updating the exe file. Grab the latest exe file

**Comment by blasco403** [comment]
Hey Gerry,

I have fat fingers all the time!!! Hahaha

Works great now after updating exe file.

Thanks for the prompt support.

TTYL

Ron

On Oct 19, 2022, at 1:33 PM, GerryDazoo ***@***.***> wrote:

﻿
I cut and pasted your line into my remote file and it worked like a champ. I checked the repository and the proper code is there.
